maxmind-geoip2-async-compile-time-di/src/main/scala/com/sovaalexandr/maxmind/geoip2/GeoIP2Components.scala

```diff
@@ -53,8 +53,14 @@ trait GeoIP2Components {
 
   protected def durationToFirstWednesdayOfNextMonth(): LocalDateTime => FiniteDuration = new DurationToFirstWednesdayOfNextMonth()
 
+  protected def databaseFileProviderSettings(configuration: Config): DatabaseFileProvider.Settings = {
+    val hostName = configuration.getString("geolocation.geoip2db.hostName")
```

Comment:
Don't think that it's a hostName. Rather persistenceId or persistenceSuffix etc.